docs/wiki/Deploying-ALZ-ZTNetwork.md

```diff
@@ -0,0 +1,49 @@
+## Azure landing zone portal accelerator deployment with Zero Trust network principles
+
+This section describes how to deploy the Azure landing zone portal accelerator with a jump start on Zero Trust Networking Principles for Azure landing zones. For more information on Zero Trust security model and principles visit [Embrace proactive security with Zero Trust](/security/business/zero-trust). Let's review the [Zero Trust aligned networking](/security/zero-trust/deploy/networks) configurations in the [Azure landing zone portal accelerator](/azure/cloud-adoption-framework/ready/landing-zone/#azure-landing-zone-portal-accelerator).
+
+## Deploy networking topology and Connectivity
+
+On the "Network Topology and Connectivity" section of the Azure landing zone portal accelerator select "Hub and spoke with Azure Firewall" radio button. Next, Select the platform connectivity subscription from the drop down. Confirm or update the address space and first networking hub region, in this case East US.
+
+![image](https://user-images.githubusercontent.com/8091766/228360733-9713f5ff-dd53-4995-b309-220442f978b5.png)
+
+Hub and spoke is the best option for Zero Trust Organizations should not have one single connection in and out of their cloud network. Network should be segmented into smaller islands with their own ingress and egress controls in minimize "blast radius". 
+
+## Enable Threat protection
+
+In the next section you will want to leave the defaults to "Enable DDoS Network Protection" and "Deploy Azure Firewall" as these are pivotal requirements for threat protection. It's also recommended to select at least two avaibility zones for Azure Firewall.  
+
+![image](https://user-images.githubusercontent.com/8091766/228363102-af09f069-c5f1-4be0-91e8-1050dc994bf9.png)
+
+These selections help to segment and enforce external and internal boundaries. 
+
+## Protect the Identity subscription
+
+On the "Identity" section, ensure the default (Yes) is selected on "Prevent inbound RDP from internet" and "Ensure subnets are associated with NSG".
+
+![image](https://user-images.githubusercontent.com/8091766/228366016-4eee4817-3885-491b-a064-7fdbaae9dc80.png)
+
+Here we are enforcing network protection on resources in the identity subscription like domain controller for example and what protocols can reach them with Network Security Groups.
```

Comment:
Not technically doing anything on rules in NSGs and therefore protocols or ports yet, so I would consider removing this as it is misleading. Just stating we enforce an NSG on the subnets in the identity sub is enough IMHO